pkg/application/options.go

```diff
@@ -243,8 +248,14 @@ func WithLoggerSettingsFromConfig(app *App) {
 		settings := &loggerSettings{}
 		config.UnmarshalKey("mon.logger", settings)
 
+		outputFile, err := os.OpenFile(settings.Output.File, os.O_CREATE|os.O_WRONLY, 0755)
```

Comment:
Should be mode 0644; Also please try whether O_APPEND works or seek to the end of the file if it is a seekable file - we would want to avoid overwriting existing logs.

Reply:
the permissions i do agree with.

For the O_APPEND, i do get this as output when trying to write to a log file on mac:
```
Failed to write to log, write output.log: bad file descriptor
```